src/Pos/Genesis.hs

```diff
@@ -221,7 +273,8 @@ devStakesDistr Nothing Nothing (Just (richs, poors, coins, richShare)) Nothing =
         then error "Impossible to make RichPoorStakes with given parameters."
         else identity
 devStakesDistr Nothing Nothing Nothing (Just n) =
-    ExponentialStakes (fromIntegral n)
+    -- minimum stake allows to have max 11 weighted stakeholders in boot era
+    ExponentialStakes (fromIntegral n) (mkCoin 2048)
```

Comment:
Again, maybe better to have this as a top-level value in this module in case it's used again.